src/components/Ring64/index.tsx

```diff
@@ -20,6 +20,22 @@ const Ring64 = ({
     {...rest} />
 );
 
+const Ring64Pair = ({
```

Comment:
`Ring64Pair` is a bit specific, as it could be used for content other than a pair. 

```suggestion
const Ring64Subtitle = ({
```
```suggestion
const Ring64Pair = ({
```

Reply:
Looks great :)